package.json

```diff
@@ -52,7 +52,8 @@
     "test": "npm-run-all dist js-test docs",
     "watch": "npm-run-all --parallel watch-css watch-js",
     "watch-css": "nodemon --ignore js/ --ignore dist/ -e scss -x \"npm run css && npm run css-docs\"",
-    "watch-js": "nodemon --ignore scss/ --ignore js/dist/ --ignore dist/ -e js -x \"npm run js-compile-plugins\""
+    "watch-js": "nodemon --ignore scss/ --ignore js/dist/ --ignore dist/ -e js -x \"npm run js-compile-plugins\"",
+    "workbox-precache": "workbox-cli inject:manifest --configFile=\"build/workbox.config.json\""
```

Comment:
workbox-precache fails on node 4 via travis as of now so I removed it from docs build, have put in a pull

It fails on [travis](https://travis-ci.org/twbs/bootstrap/jobs/265776270) due to incompatibility with node 4, so I had to remove it from the docs script to pass, but i've put in a [pull request](https://github.com/GoogleChrome/workbox/pull/753/files) to fix it though, so waiting on that for now